gobblin-service/src/main/java/org/apache/gobblin/service/modules/core/GobblinServiceManager.java

```diff
@@ -265,7 +266,9 @@ public static GobblinServiceManager create(GobblinServiceConfiguration serviceCo
    */
   public static <T> T getClass(Class<T> classToGet) {
     if (GOBBLIN_SERVICE_GUICE_MODULE == null) {
-      throw new RuntimeException("getClass called without calling create method to initialize GobblinServiceGuiceModule");
+      throw new RuntimeException(String.format("getClass called to obtain %s without calling create method to "
+          + "initialize GobblinServiceGuiceModule. Stacktrace of current thread %s", classToGet,
+          Arrays.toString(Thread.currentThread().getStackTrace()).replace(", ", "\n  at ")));
```

Comment:
sorry if my comment was confusing, but the `RuntimeException` should generate a stacktrace already